maquetado/setup/scss/tech.scss

```diff
@@ -8,18 +8,24 @@
   height: 300px;
   display: flex;
   flex-direction: column;  
-  align-items: center;
-
+  align-items: center;  
+  
   &::before {
     content: '';
     border: 2px solid  $green;
     position: absolute;
-    right: 5px;
-    bottom: 5px;
+    left: -7px;
+    top: -7px;
     z-index: -1;
     border-radius: 10px;
     width: 210px;  
     height: 300px;
+    transition: left 1s, top 1s;  
```

Comment:
You should add a transition effect in here, `ease` for example

`transition: top 1s ease, left 1s ease;`